framework/src/st_things/things_stack/cloud/cloud_manager.c

```diff
@@ -654,6 +654,8 @@ OCStackApplicationResult handle_login_cb(void *ctx, OCDoHandle handle, OCClientR
 			return OC_STACK_DELETE_TRANSACTION;
 		}
 
+		esm_save_easysetup_state(1);
```

Comment:
Would you like to use definitions?
I don't realize the meaning of '1'.
Ex)
#define ES_SUCCESS 1
#define ES_FAIL 0

Reply:
It is good to your opinion.